<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginscreen</class>
 <widget class="QDialog" name="loginscreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1012</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Yoobe Library System | Login</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/images/favicon.ico</normalon>
   </iconset>
  </property>
  <widget class="QFrame" name="gridFrame_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>801</width>
     <height>551</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
background: rgb(255, 255, 255);
font-size: 15px;
font-family: Century Gothic, sans-serif;
}

QFrame{
border: solid 10px rgba(0, 0, 0);
}

QLineEdit{
color: #8d98a1;
background-color: #d3d3d3;
font-size: 18px;
border-style: outset;
border-radius: 10px;
}
QPushButton{
background: #ced1d8;
border-style: outset;
border-radius: 5px;
}
QPushButton:pressed{
background-color: rgb(224, 0, 0);
border-style: inset;
}
</string>
   </property>
   <widget class="QLabel" name="img_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>341</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Yoobee image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="close">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>60</y>
      <width>172</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
  color: white;
  background-color: #27a9e3;
  border-width: 0px;
  border-radius: 3px;
}
hover:

QPushButton:hover { background-color: #66c011; }</string>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Create">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>410</y>
      <width>256</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
  color: white;
  background-color: #27a9e3;
  border-width: 0px;
  border-radius: 3px;
}
hover:

QPushButton:hover { background-color: #66c011; }</string>
    </property>
    <property name="text">
     <string>Create An Account</string>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>241</y>
      <width>221</width>
      <height>162</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Username</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="username_input">
       <property name="styleSheet">
        <string notr="true">QLabel { color: white; }
QLineEdit { border-radius: 3px; }</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="password_input">
       <property name="styleSheet">
        <string notr="true">QLabel { color: white; }
QLineEdit { border-radius: 3px; }</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="login">
       <property name="styleSheet">
        <string notr="true">QPushButton
{
  color: white;
  background-color: #27a9e3;
  border-width: 0px;
  border-radius: 3px;
}
hover:

QPushButton:hover { background-color: #66c011; }</string>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_rememberMe">
       <property name="text">
        <string>Remember me</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>username_input</tabstop>
  <tabstop>password_input</tabstop>
  <tabstop>login</tabstop>
  <tabstop>checkBox_rememberMe</tabstop>
  <tabstop>Create</tabstop>
  <tabstop>close</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
